integration tests fail to build with CMake 3.26

Hi,

I am unable to build the integration tests on macOS. Here is the full output:

```
(lp) ➜  integration_tests git:(main) ./run_tests.py
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-llvm
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-c
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-wasm
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-cpython
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-x86
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-wasm_x86
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-wasm_x64
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-c_py
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-c_sym
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-cpython_sym
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-llvm_sym
+ rm -rf /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-llvm_py
+ mkdir /Users/benwibking/lpython/integration_tests/_lpython-tmp-test-llvm
+ cmake -DKIND=llvm -DFAST=no -DPYTHON_LIBS_REQ=no ..
-- The C compiler identification is AppleClang 14.0.3.14030022
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done


Configuration results
---------------------
C compiler      : /Library/Developer/CommandLineTools/usr/bin/cc
Build type: Debug
C compiler flags      : -g
Installation prefix: /usr/local
KIND: llvm
FAST: no
PYTHON_LIBS_REQ: no
LPYTHON: /Users/benwibking/lpython/src/bin/lpython
LPYTHON_RTLIB_DIR: /Users/benwibking/lpython/src/libasr/runtime
LPYTHON_RTLIB_LIBRARY: /Users/benwibking/lpython/src/runtime/liblpython_runtime_static.a
-- Configuring done (0.4s)
CMake Error: CMake can not determine linker language for target: import_order_01
-- Generating done (0.1s)
CMake Generate step failed.  Build files cannot be regenerated correctly.
Command failed: cmake -DKIND=llvm -DFAST=no -DPYTHON_LIBS_REQ=no ..
```

--- a/integration_tests/CMakeLists.txt
+++ b/integration_tests/CMakeLists.txt
@@ -375,6 +375,7 @@ macro(COMPILE)
                 DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
                 VERBATIM)
             add_library(${name} OBJECT ${name}.o)
+            set_target_properties(${name} PROPERTIES LINKER_LANGUAGE C)
         elseif(KIND STREQUAL "c")
             add_custom_command(
                 OUTPUT ${name}.c
@@ -382,6 +383,7 @@ macro(COMPILE)
                 DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
                 VERBATIM)
             add_library(${name} OBJECT ${name}.c)
+            set_target_properties(${name} PROPERTIES LINKER_LANGUAGE C)
             target_link_libraries(${name} lpython_rtlib)
         elseif(KIND STREQUAL "cpython")
             # CPython test

--- a/integration_tests/run_tests.py
+++ b/integration_tests/run_tests.py
@@ -25,7 +25,7 @@ def run_test(backend):
     run_cmd(f"mkdir {BASE_DIR}/_lpython-tmp-test-{backend}", cwd=BASE_DIR)
     cwd = f"{BASE_DIR}/_lpython-tmp-test-{backend}"
     run_cmd(f"cmake -DKIND={backend} -DFAST={fast_tests} -DPYTHON_LIBS_REQ={python_libs_req} ..", cwd=cwd)
-    run_cmd(f"make -j{DEFAULT_THREADS_TO_USE}", cwd=cwd)
+    run_cmd(f"cmake --build . --parallel {DEFAULT_THREADS_TO_USE}", cwd=cwd)
     run_cmd(f"ctest -j{DEFAULT_THREADS_TO_USE} --output-on-failure",
             cwd=cwd)
 

